### ISSUE TITLE
Failing integration tests DocumentationTest

### ISSUE THREAD

**Issue by kche309** [bioDS/beast-phylonco#9]
Building package fails beast integration test with the error below
```
junit_phylonco:
    [mkdir] Created dir: /home/kylie/Repos-new/beast-phylonco/build/junitreport
    [junit] Running test.beast.integration.DocumentationTest
    [junit] Tests run: 3, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.763 sec
    [junit] Test test.beast.integration.DocumentationTest FAILED
```

@kche309 to fix DocumentationTest

**Comment by J-Moravec** [comment]
Seems like what is being tested is `BEAST2` code, not `phylonco` code. Integration tests might not be properly set up?
![image](https://user-images.githubusercontent.com/10236938/152274461-749f8922-8c9a-4e36-b25b-8bb7923d91a9.png)

**Comment by kche309** [comment]
I agree, it looks like the failing classes are from `BEAST2` and other packages rather than the `phylonco` package. It is strange it is not picked up in the beast2 github continuous integration workflow (e.g. https://github.com/CompEvol/beast2/runs/4925469365?check_suite_focus=true). 

On further inspection, it seems only beast-core tests are run for `BEAST2` CI

**Comment by walterxie** [comment]
If you did not change any Java classes, I used the Gradle to run all tests just now and they were passed:

<img width="1169" alt="Screen Shot 2022-02-04 at 12 51 06 PM" src="https://user-images.githubusercontent.com/5743014/152448540-33095afa-9353-4a13-8793-7e4e811dabc8.png">

**Comment by J-Moravec** [comment]
@walterxie This is about integration tests, not unit tests.

**Comment by kche309** [comment]
Ant builds are no longer used, this is being replaced with Gradle. 
Integration tests will be setup on Docker similar to https://github.com/tgvaughan/bacter 

TODO: @kche309 to set up Docker file for integration tests

**Comment by kche309** [comment]
I've had a quick look into Gradle documentation, setting up integration tests looks a bit tricky https://docs.gradle.org/current/userguide/java_testing.html#sec:configuring_java_integration_tests 
Beast2 does not package tests in the jar, so we can't load integration tests directly from the jar. 
Probably it is easier to have a Docker + ant setup that can run and test in a clean environment.
